docs/jetstream/operations.md

```diff
@@ -0,0 +1,129 @@
+```
+
+The dashboard can than be accessed via [127.0.0.1:8080](http://127.0.0.1:8080) through the web browser and provides a detailed overview of past workflows, the statuses of each step in a workflow and container logs.
+
+### How do I know if something went wrong?
+
+Jetstream logs errors to the console and, optionally, to the `monitoring.logs` BigQuery table. Logging to BigQuery is enabled by default when running Jetstream via Airflow, as it allows for better alerting and monitoring of errors. It is by default disabled for runs triggered via the jetstream-config CI.
+
+Errors can be viewed on the [Jetstream error dashboard] in Redash.
+
+Additionally, alerts have been set up in Redash to check for errors daily and sent an email if failures have been detected. To subscribe to these alerts, email addresses can be added as _Destinations_ in the [Redash alert configuration](https://sql.telemetry.mozilla.org/alerts/81).
+
+### Something went wrong, what do I do?
+
+1. Check the [Jetstream error dashboard] for more details on the error that occurred.
+1. If the experiment uses a custom configuration, make sure the configuration is valid. Sometimes, SQL written for specifying metrics in the configuration file can contain logical errors that result in failures when computing statistics.
+    * It is possible that SQL that was once valid starts to fail, because the query has become too complex. This can happen if new fields get added to a table that is being queried. In these cases, try to rewrite and simplify the SQL query or consider using source tables instead of derived views. 
+
+If you are unsure of what might have gone wrong or what to, you can open [an issue in Jira](https://jira.mozilla.com/projects/CIRRUS/issues/CIRRUS-68?filter=allopenissues) or ask for help in the [#ask-experimenter](https://mozilla.slack.com/archives/CF94YGE03) Slack channel.
+
+### Some results appear to be missing
+
+It can take up to 2 days for results of the overall time period to be available after an experiment ends. For example, if an experiment ends on 2021-04-05, then results for the overall time period will be computed in the next daily analysis run on 2021-04-06. As running the analysis can take a few hours to complete results will be available on 2021-04-07.
+
+If results other than for the overall period are missing or more than 2 days have passed and no overall results are available, check the [Jetstream error dashboard] if there are any analysis errors for your experiment.
+
+If there have been no errors, or the errors cannot be resolved, open [an issue in Jira](https://jira.mozilla.com/projects/CIRRUS/issues/CIRRUS-68?filter=allopenissues) or ask for help in the [#ask-experimenter](https://mozilla.slack.com/archives/CF94YGE03) Slack channel.
+
+### How do I debug operational or more complex errors?
```

Comment:
This is not an exhaustive list of things that could go wrong but maybe still helpful to have a list of more common issues that could be checked first.